src/contract.ts

```diff
@@ -16,7 +15,7 @@ interface ContractBuild {
 }
 
 export function getBuildFile(build_dir: string, trace?: boolean): string {
-  const fileName = trace ? `contract-trace.json` : `contract.json`;
+  const fileName = trace ? `contract-trace.json` : `contracts.json`;
```

Comment:
decent chance this breaks open oracle or w/e other project this was being developed with, assuming contract.json works in at least one situation?

Reply:
investigating.. though looks like open oracle also uses `contracts.json`

Reply:
guess just a typo all along

Reply:
actually, this typo just got introduced in [0.1.6](https://github.com/compound-finance/saddle/commit/da9118d0ccfa785e4e4b83022542d7d89089057d#diff-bb179d2c3824efa379684e18610b93a3)

Reply:
actually, it looks like open-oracle does use `contract.json` in the docker builds. cant really decipher why (or how). @hayesgm ?

Reply:
Yeah, we were awkwardly assuming the build_dir was `./build` in places, even though it's actually specified in the settings. I fixed that in 0.1.6, but I guess I fixed it to be `contract.json` not `contracts.json`. We can make this fix, but we should also name this `contracts-trace.json` too, then, right?